config/locales/en/nsm/work_items.yml

```diff
@@ -30,6 +30,11 @@ en:
         allowed_cost: Net cost allowed
         total: Total
         table_info_item: work item
+        summary_table_caption: Summary of claimed work items
+        table_caption: Claimed work items
+        accessible:
+          claimed: 'Sum of net cost claimed: '
```

Comment:
since this is read by screen readers should it say "Sum of net cost claimed/allowed is"?

Reply:
worth noting not all screen readers will read the slash so "claimed or allowed" might be better? But maybe we can have it like this for now, ask content and change later?

Reply:
so in this instance it is only claimed... but this is more a question for chris/mary I would expect...